pandas/core/groupby.py

```diff
@@ -233,6 +233,8 @@ class Grouper(object):
 
     >>> df.groupby(Grouper(level='date', freq='60s', axis=1))
     """
+    _attributes = (('key', None), ('level', None), ('freq', None),
```

Comment:
again you don't need to specify the default, just list the attributes. The repr should include all the values, not just the defaults. Further this means you are defining defaults in 2 places, here and the init, not a good idea.

Reply:
Ok, changed. Also added test for ``TimeGrouper.__repr__``. 

Should a note about this go in the 0.21.1 or 0.22 whatsnew.rst?